.github/workflows/ci.yml

```diff
@@ -37,13 +38,16 @@ jobs:
 
   PHPUnit-hhvm:
     name: PHPUnit (HHVM)
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
Using `ubuntu-22.04` makes perfect sense :+1: 

It appears that we're still using the old `ubuntu-20.04` in our PHPUnit job within the `ci.yml` file. I believe this should also be updated to `ubuntu-22.04`, just like you did for the PHPUnit-hhvm job.